### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by Havouza** [thirtybees/tbupdater#1]
Hi!
The install OK but update dont work. Every step produce the attached warning and after pressing update, nothing happens, only the warnings
![screenshot 2017-04-26 19 33 12](https://cloud.githubusercontent.com/assets/25871096/25447487/0eb63854-2abe-11e7-9cff-4b499a8fc8fd.png)

**Comment by Havouza** [comment]
Installed 1.1.1

You do not have permission to configure this module.

**Comment by firstred** [comment]
Confirmed. Thanks

**Comment by firstred** [comment]
But does it block you from upgrading? I just successfully upgraded 3 stores that showed this warning, too.

**Comment by SiliticX** [comment]
http://c2.acolyte.ws/Screenshot-at-14-04-09.png

having the same issue i believe.

**Comment by SeeTwoDev** [comment]
Having the Notices in debug mode, too.

And i can't seem to go forward. the script seems to think not all the checkboxes are there. 
![screenshot-at-14-04-09](https://cloud.githubusercontent.com/assets/5693643/25449441/bf588c3c-2a89-11e7-90ac-0b56b6c3e7cf.png)

**Comment by firstred** [comment]
There is one check missing from the list that is being displayed, I noticed and that is "is the autoupgrade folder (admin/autoupgrade) writable?"

**Comment by SeeTwoDev** [comment]
Check gitter, think i found the issue

**Comment by Havouza** [comment]
The module does not get installed even it it say so. It is still in the list with green install button

**Comment by SeeTwoDev** [comment]
Got it to work.

Change lin 864 in tbupdater.php to $allowedArray['admin_au_writable'] = ConfigurationTest::testDir($adminDir.DIRECTORY_SEPARATOR.$tools->autoupgradeDir, false, $report, false);

(the false at the end was change from true. It's a relative path, not asolute.

**Comment by firstred** [comment]
@Havouza What permissions does your current employee have and what version are you using?

**Comment by Havouza** [comment]
Superadmin and TB1.0.0 updater 1.1.1

**Comment by Havouza** [comment]
That change did not solve my problem. I still cant install the module even if it say it is
![screenshot 2017-04-26 20 52 29](https://cloud.githubusercontent.com/assets/25871096/25450953/40dec656-2ac9-11e7-9784-5ad0c99898cb.png)

**Comment by firstred** [comment]
@Havouza Same problem with overrides disabled?

**Comment by Havouza** [comment]
I deleted the module and uploaded it again, changed the line 864 and the update works but get the same errors as SiliticX

So I have restored latest backup from this morning and will test again tomorrow. Perhaps the patches are correct then. For the moment I see double computers.

Thanks for amazing work

**Comment by Havouza** [comment]
Hi Michael!



Cant add this in Github somehow


Well, slept on it and made a new test today. One warning came up on install but install run

 

After update following warning

```

 

`SQL 1.0.1 1146 in ALTER TABLE `tb_product_shop_attribute` ALTER COLUMN `available_date` SET DEFAULT '1970-01-01': Table 'db_beauty.tb_product_shop_attribute' doesn't exist

SQL 1.0.1 1146 in UPDATE `tb_product_shop_attribute` SET `available_date` = '1970-01-01' WHERE CAST(`available_date` AS CHAR(20)) = '0000-00-00 00:00:00': Table 'db_beauty.tb_product_shop_attribute' doesn't exist`

```

Warning detected during upgrade.

 

When I look in the db the name of the table is product_attribute_shop

**Comment by firstred** [comment]
Initial issue has been addressed with version 1.2.5. If you find anything new feel free to open a new issue.
